controlsfx/src/main/java/org/controlsfx/control/TaskProgressView.java

```diff
@@ -62,7 +64,7 @@
  */
 public class TaskProgressView<T extends Task<?>> extends ControlsFXControl {
 
-    private boolean retainTaskMode = false;
+    private BooleanProperty retainTasks = new SimpleBooleanProperty(this, "retainTasks", false);
```

Comment:
You also forgot the property accessor method "retainTasksProperty()" (also final). And in JavaFX controls there is this convention to keep the field and its three methods (xyzProperty(), isXyzProperty(), setXyzProperty()) grouped together in the source code.

Reply:
- `retainTasksProperty()` seems there already
- `setXyzProperty()` doesn't make sense since the property has been made `final` right?